subsys/fs/shell.c

```diff
@@ -33,6 +33,16 @@ static struct fs_mount_t fatfs_mnt = {
 /* LITTLEFS */
 #ifdef CONFIG_FILE_SYSTEM_LITTLEFS
 #include <zephyr/fs/littlefs.h>
+
+#ifdef CONFIG_FS_LITTLEFS_BLK_DEV
+FS_LITTLEFS_DECLARE_DEFAULT_CONFIG(lfs_data);
+static struct fs_mount_t littlefs_mnt = {
+	.type = FS_LITTLEFS,
+	.fs_data = &lfs_data,
+	.flags = FS_MOUNT_FLAG_USE_DISK_ACCESS,
+	.storage_dev = CONFIG_MMC_VOLUME_NAME,
```

Comment:
it would be cool to have an opportunity to enter this name from shell in order to use different devices in runtime, like "MMC", "SD", "SD1" or "MMC1", but l see no reason to do it now